pkgs/repo_manage/lib/issue_transfer.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:github/github.dart';
+import 'package:graphql/client.dart';
+
+import 'src/common.dart';
+
+class TransferIssuesCommand extends ReportCommand {
+  TransferIssuesCommand()
+      : super('transfer-issues',
+            'Bulk transfer issues from one repo to another.') {
+    argParser.addFlag(
+      'apply-changes',
+      negatable: false,
+      help: 'WARNING: This will transfer the issues. Please preview the changes'
+          "first by running without '--apply-changes'.",
+      defaultsTo: false,
+    );
+    argParser.addMultiOption(
+      'issue-numbers',
+      valueHelp: '1,2,3',
+      help: 'Specifiy the numbers of specific issues to transfer, otherwise'
+          ' transfers all.',
+    );
+    argParser.addOption(
+      'source-repo',
+      valueHelp: 'repo-org/repo-name',
```

Comment:
also 'mandatory'?

Reply:
To be clear, I don't know if the error message you'd get out of the argparser here is as good as the manual one you have below.

Reply:
Still makes sense I think, especially for clearer code